src/inc/schedule-output-functions.php

```diff
@@ -436,25 +440,23 @@ function wpcs_scheduleOutput( $props ) {
 							$content .= sprintf( ' <span class="wpcs-session-speakers">%s</span>', esc_html( $speakers ) );
 						}
 
-						// Session Content Footer Filter
+						// Session Content Footer Filter.
 						$wpcs_session_content_footer = apply_filters( 'wpcs_session_content_footer', $session->ID );
-						$content                    .= ( $wpcs_session_content_footer != $session->ID ) ? $wpcs_session_content_footer : '';
+						$content                    .= ( $wpcs_session_content_footer !== $session->ID ) ? $wpcs_session_content_footer : '';
 
 						// End of cell-content.
 						$content .= '</div>';
 
 						$columns_clone = $columns;
 
-						// If the next element in the table is the same as the current one, use colspan
-						if ( $key != key( array_slice( $columns, -1, 1, true ) ) ) {
-							// while ( $pair = each( $columns_clone ) ) {
-							// foreach($columns_clone as $pair) {
+						// If the next element in the table is the same as the current one, use colspan.
+						if ( key( array_slice( $columns, -1, 1, true ) ) === $key ) {
```

Comment:
Was this an intentional logic change? Your change matches the comment, but doesn't match the previous code.

Reply:
Thanks for confirming! Agreed; logic changes should be handled separately. I've added an issue to follow up on this.